charts/drax/values.yaml

```diff
@@ -819,6 +836,46 @@ influxdb:
         CREATE DATABASE "db_5G" WITH DURATION 5d REPLICATION 1 NAME "rp_5d_5G"
 
 
+influxdb2:
+  enabled: true
+
+  podLabels:
+    drax/role: ric
+    drax/component-name: influxdb2
+
+  adminUser:
+    organization: "accelleran"
+    bucket: "default"
+    retention_policy: "0s"
+    user: "admin"
+    password: ""
+    token: ""
+    existingSecret: drax-influxdb2-auth
```

Comment:
We should open a PR at the influxdb2 chart to allow injecting the release name then. Good enough as it is though to get this merged and start using it.